API.md

```diff
@@ -648,12 +681,19 @@ Has the same properties as a [TextField](#textfield-component), with additional:
   - `options`:
     - `autocomplete` - boolean, when `true` will set the autocomplete attribute on so that users can easily add their
       own telephone number.
+    - `prefix` - optional prefix for the text field.
+    - `suffix` - optional suffix for the text field.
 
 ## `TextField` component
 Simple text field.
   - `schema`:
     - `max` - number - the maximum length of text to allow.
     - `trim` - boolean - whether to force whitespace trimming from the start and end of the text.
+  - `options`:
+    - `prefix` - optional prefix for the text field.
+    - `suffix` - optional suffix for the text field.
+
+Some users may miss that the input already has a suffix or prefix, and enter a prefix or suffix into the input. Allow for this in your validation and do not show an error.
```

Comment:
I think this approach is fine for now.

If we want to introduce built-in validation handling for when a user includes a prefix/suffix in the value entered then we can do that as an additional enhancement later.